app/resources/styles.tsx

```diff
@@ -46,4 +52,21 @@ export const DropdownContainer = styled.div`
     font-family: "Poppins";
     font-weight: 500;
     margin: 0;
-    `
\ No newline at end of file
+    `
+    export const VideoCard  = styled.div <Props>`
+    width: 250px;
+    height: 250px;
+    background-color: #D1D1D1;
+    border-radius: 20px;
+    background-image: url(${props => props.image || ''});
```

Comment:
Is the empty '' about the image issues you had?

Reply:
The image link its supposed to be in the empty commas